src/engine/V3Eval.v3

```diff
@@ -835,6 +866,45 @@ def do_vv_v_x16(a: (u64, u64), b: (u64, u64), f: (u8, u8) -> u8) -> (u64, u64) {
 	}
 		return (low, high);
 }
+// Apply a binary operation to adjacent lanes and put the result into a wider lane
+def do_v_v_x16_pairwis_ext_x8(a: (u64, u64), f: (u8, u8) -> u16) -> (u64, u64) {
+    var low: u64 = 0;
+    var high: u64 = 0;
+    
+    for (shift: byte = 0; shift < 64; shift += 16) {
+        var r_a1 = u8.view((a.0 >> shift) & 0xFF);
```

Comment:
You can also just do `u16.view(u8.view(x))` and then add/sub/multiply using, e.g. `u16.+`.